Assets/Feature01/Scripts/Controller/PlayerController.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerController : MonoBehaviour
+{
+    Rigidbody _rigidbody;
+    [SerializeField] PlayerOnFootInput playerFootInput;
+    [SerializeField] PlayerInput playerInput;
+    [SerializeField] MovementController movementController;
+    [SerializeField] GamePadCursor gamepadCursor;
+
+    Vector3 playerMoveInput;
+    //Vector3 playerLookInput;
+
+
+    public Vector2 movementInput;
+
+    public Vector2 getDirection;
+    public Vector3 cursorPosition;
+
+ 
+
+
+
+    [SerializeField] float playerSpeed = 30.0f;
+    [SerializeField] float _cursorSpeed = 30f;
+
+    private void Awake()
+    {
+        _rigidbody = GetComponent<Rigidbody>();
+     
+        
+    }
+
+
+    private void FixedUpdate()
```

Comment:
This is not how you should use New input system. You are basically using the same paradigm as old input system. I recommend watching a few tutorials.

How it should work is . whenever your stick is moving something is happening. Event driven. Not Update Driven.

Reply:
bhai ami kiso tutorial dekhsilam,  oigula te emon fixed update er vitor  diye kora, event driven input  niye constantly check kore. 
bhai any recommendation on tutorial jekhane only event fire korle e move hocche